Fix message when user hits max credits

kennnyshiwa opened the pull request:
### Type

- [X] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes
Fixes the message a user should get when they have max credits and attempt to use the payday command

Commit:
Fix message when user hits max credits

Fixes the error message when a users issues the payday command when having max credits

mikeshardmind reviewed (dismissed):
Side note: We have some inconsistent breakage in #testing with this.

kennnyshiwa committed:
Update economy.py

Changed message when user hits max payday and bank is global to match message when bank is per server

kennnyshiwa committed:
Update economy.py

made statements match
